static/js/modules/set-task-state.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function(task, state) {
```

Comment:
I believe this is another example of copied code

Reply:
Added comments everywhere this is used.